### ISSUE TITLE
Scene : Make the versioning system functional

### ISSUE THREAD

**Issue by neomonkeus** [niftools/blender_nif_plugin#232]
Implement an actual versioning system which is user friendly.

Currently we store the version information in the scene for comparability reasons.
I was originally going to totally remove it although as it doesn't add value and its the main generator of issues.

Existing solution was based on the fact that we only really care about versioning at import/export time.

Reasoning 
- Reduce export ui
- Decouple the code in some areas that relies on version information
- Information Filtering - https://github.com/niftools/blender_nif_plugin/issues/59

Issues with solution : 
- Knowing whether or not the global should be updated
- Conversion between decimal and hex
- Different systems to represent the same information variant - game, version, tuples

This is an usability improvement over the following basic implementation #227

Improvements to solutions
- [x] Let the user decide to override the target on import
- [x] Decouple the versioning code
- [ ] Implement a user friendly ui that exposes the more common x.y.z notation
- [ ] store a user friendly version
- [ ] Game prefabs

**Comment by Dexesttp** [comment]
This seems to be a duplicate of #227 , but with a broader view and a better improvement list. I'm therefore proposing to close #227 .

**Comment by neomonkeus** [comment]
We can keep the original and use this as a summary issue for the full implementation. The other issue can finished in isolation but still adds to this overall implementation.

**Comment by syscrusher1709** [comment]
I just encountered this set of issues (specifically, #227, but as you point out they are closely inter-related). If I am not mistaken, shouldn't the choice of which game to import/export imply the NIF versions?

**Comment by neomonkeus** [comment]
tldr; good idea, unusable implementation

There are pros and cons to both. Ironically a lot of functionality to help improve usability that we want to implement will rely on knowing the version information before export, i.e exposing version specific options, eg bsxflag, collision, layers information, etc. which going with original export time version solutions is not available, so you have to cram the UI there.

The goal of this summary issue to make the system both usable and user-friend. 
Ideally the system should never have been implemented until it was needed nor should it have been done the way it was, but that is the current position. The result is more work to rip out the bits we don't want/need and move towards it being usable and then user friendly. This issue has highest priority as

**Comment by neomonkeus** [comment]
Initial part of the implementation is integrated to develop branch. You can now set the global version on the scene or choose at export time from a range of games as per previous implementation.
The scene level setting exposed via the UI are not currently use friendly as you need to know the corresponding integer values, so next step would be to have prefabs for each supported game.

**Comment by HENDRIX-ZT2** [comment]
What I have in mind right now is moving the game dropdown over from the export op to the scene version panel. Then bind some logic to it so that the numerical version is updated if the user changes the game prefab. And export only fetches the final numerical version from the scene panel.
